addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -105,8 +105,16 @@ var SnippetEditor = Widget.extend({
         }
 
         var _animationsCount = 0;
-        var postAnimationCover = _.throttle(() => this.cover(), 100);
-        this.$target.on('transitionstart.snippet_editor, animationstart.snippet_editor', () => {
+        var postAnimationCover = _.throttle(() => {
+                if (_animationsCount > 0) {
+                    _animationsCount--;
+                }
+                this.$el.toggleClass('d-none', _animationsCount > 0);
+                this.cover();
+            },
+            100
+        );
```

Comment:
I don't think this is right. If we start decreasing the animation count in that method and that this method is called both after a setTimeout and the transitionend event, then the counter will be lowered by 2 instead of 1 for each animation. Sure it would work when one transition but...

Probably not the best code I made since the rush but the code of `_updateMainPaddingTopLoop` is pretty similar to what we want here and I think it handles it correctly.

Reply:
Ok have done so.